### ISSUE TITLE
Error when upgrading to laravel 5.6

### ISSUE THREAD

**Issue by CrixuAMG** [bugsnag/bugsnag-laravel#286]
```
PHP Fatal error:  Interface 'Illuminate\Contracts\Logging\Log' not found in /home/vagrant/dev/dealmaker-api/vendor/bugsnag/bugsnag-laravel/src/LaravelLogger.php on line 12

In LaravelLogger.php line 12:
                                                          
  Interface 'Illuminate\Contracts\Logging\Log' not found  
```

As stated in the title, using Laravel 5.6, upgrading from 5.5.
PHP version: 7.1.6.

**Comment by Cawllec** [comment]
Hi @CrixuAMG, @GrahamCampbell is currently working on an update to work with 5.6, this should be available soon.

**Comment by browner12** [comment]
The `Illuminate\Contracts\Logging\Log` interface has been removed, so one thing you'll have to do is update your `AppServiceProvider.php` (or wherever you put this code), and remove 

```
$this->app->alias('bugsnag.multi', \Illuminate\Contracts\Logging\Log::class);
```

**Comment by GrahamCampbell** [comment]
The actual bugsnag class implements the interface that no longer exists, so there will be more work than that required. I am working on the upgrade over the next few days.

**Comment by browner12** [comment]
ok, thanks.

**Comment by georgewritescode** [comment]
@GrahamCampbell Do you have an estimated time until fix? 
It's holding up my deployment at the moment.

**Comment by GrahamCampbell** [comment]
The example app shows how to install the fork via composer: https://github.com/GrahamCampbell/bugsnag-laravel/blob/34d75132a7747d013049fd9cfa5683a63be70db1/example/laravel56/composer.json.

**Comment by mattdfloyd** [comment]
With the fork, it appears that setting up bugsnag as a logging endpoint (instead of aliasing the log interfaces) is the other relevant change . Anything else @GrahamCampbell ?

https://github.com/GrahamCampbell/bugsnag-laravel/blob/34d75132a7747d013049fd9cfa5683a63be70db1/example/laravel56/config/logging.php

**Comment by GrahamCampbell** [comment]
Yes, I think that is all that is needed. I am writing some documentation at the moment.

**Comment by kattrali** [comment]
Fixed in [v2.14.0](https://github.com/bugsnag/bugsnag-laravel/releases/tag/v2.14.0)

The new integration instructions are [here](https://docs.bugsnag.com/platforms/php/laravel/#laravel-5-6)

**Comment by mhimi1** [comment]
Hi,

This also breaks in "laravel/lumen-framework": "5.6.*",
